ticket.go

```diff
@@ -9,11 +12,13 @@ type Ticket struct {
 	ExternalID          *string             `json:"external_id,omitempty"`
 	Type                *string             `json:"type,omitempty"`
 	Subject             *string             `json:"subject,omitempty"`
+	Comment             *string             `json:"comment,omitempty"`
 	Description         *string             `json:"description,omitempty"`
 	Priority            *string             `json:"priority,omitempty"`
 	Status              *string             `json:"status,omitempty"`
 	Recipient           *string             `json:"recipient,omitempty"`
 	RequesterID         *float64            `json:"requester_id,omitempty"`
+	Requester           *Requester          `json:"requester_id,omitempty"`
```

Comment:
1. *User instead of *Requester
2. `json:"requester,omitempty"` instead of `json:"requester_id,omitempty"`